### PR TITLE
Add San Diego and Sao Paulo to Master 

### LINKED ISSUE #5: GitHubTeacher - San Diego, CA
## San Diego, CA

- [x] Create a branch.
- [x] Add File
- [x] Create a commit.
- [x] Open a PR
- [ ] Have a discussion about PR
- [ ] Merge PR

### PR DESCRIPTION
**Pull request by githubteacher**
Closes #5

### CODE REVIEW

**Comment by githubteacher** [comment]
@beardofedu What do you think about my changes?

File: places/adannnsergio-saopaulo.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+### Fun Facts
+* There is a state and a city with the same name. The city, in this case, is the state capital.
+
+* The city of Sao Paulo is known as the "city of drizzle" because almost every day it rains in the evening.
+
+* The state of Sao Paulo is the richest state in Brazil, there is a cultural concentration, industrial and business. But it is not perfect, our police is violent and there is much social inequality.
+
+### Places to Eat
+* Vegg's Restaurant Vegetarian and Vegan
```
Comment (saugup) [inline places/adannnsergio-saopaulo.md]: Comments from saugup

**Comment by beardofedu** [comment]
I think these changes 👀 great! 🚢 🚢 :shipit:

**Comment by adannsergio** [comment]
Great!

File: places/githubteacher-sandiego.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+## San Diego, CA
+
+### Fun Facts
+- It is the primary location of the movie "Top Gun".
+
+### Places to Eat
+- City Tacos
+
+### Places to Go
+- Balboa Park
```
Comment (grandpaklaus) [inline places/githubteacher-sandiego.md]: I studied for a year at a college in El Cajon and visited Balboa Park.

**Pull request merged**